spec/system/user_creates_note_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "user creates a note", js: true do
+  context "when the current user creates a note" do
+    include_context "user in an active call"
+
+    describe "creating a new note" do
+      it "formats the user's first name, a colon, and then their note" do
+        fill_in "note_body", with: "this is my note"
+        click_on "Create Note"
+        expect(page).to have_content /Elon:\s+this is my note/
```

Comment:
I'm curious why you used regex here instead of matching a string. Does a string not work here for some reason?

Reply:
Capybara returns `Elon:\n this is my note` as part of the `page` content. I saw this start to happen when I used`&nbsp;` in the HTML. That newline character causes the test to fail if the pattern to match is just a string.